Error and Warning message at running display.launch

![error_message](https://user-images.githubusercontent.com/18494952/75009243-89763900-54bd-11ea-8770-ca65d0f95fc9.png)

```
[WARN] [1582788547.537572]: The 'use_gui' parameter was specified, which is deprecated.  We'll attempt to find and run the GUI, but if this fails you should install the 'joint_state_publisher_gui' package instead and run that.  This backwards compatibility option will be removed in Noetic.
[ERROR] [1582788547.538605]: Could not find the GUI, install the 'joint_state_publisher_gui' package
[ WARN] [1582788547.556383516]: The 'state_publisher' executable is deprecated. Please use 'robot_state_publisher' instead
```

--- a/launch/display_urdf.launch
+++ b/launch/display_urdf.launch
@@ -8,7 +8,7 @@
   <param name="robot_description" textfile="$(arg model)" />
   <param name="use_gui" value="$(arg gui)" />
   <!-- nodes -->
-  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+  <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
   <!-- rviz -->
   <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

--- a/launch/display_xacro.launch
+++ b/launch/display_xacro.launch
@@ -7,7 +7,7 @@
   <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
   <param name="use_gui" value="$(arg gui)" />
 
-  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+  <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
   <node name="rviz" pkg="rviz" type="rviz" required="true" args="-d $(arg rvizconfig)" />
 </launch>

--- a/package.xml
+++ b/package.xml
@@ -18,6 +18,7 @@
   <build_depend>urdf</build_depend>
 
   <run_depend>urdf</run_depend>
+  <run_depend>joint_state_publisher_gui</run_depend>
 
   <export>
 

